app/models/bgs_power_of_attorney.rb

```diff
@@ -192,6 +191,10 @@ def person
     @person ||= Person.find_or_create_by_participant_id(poa_participant_id)
   end
 
+  def related_appeals
+    claimants.select(decision_review_type: "Appeal")
```

Comment:
Should this be something like:
```suggestion
    claimants.select { |c| c.decision_review_type == "Appeal" }
```
Also, this only gets the claimant. We don't have the appeal yet.